src/main/java/org/prebid/server/bidder/beachfront/BeachfrontUsersyncer.java

```diff
@@ -12,15 +13,21 @@
 
     private final UsersyncInfo usersyncInfo;
 
-    public BeachfrontUsersyncer(String usersyncUrl, String platformId) {
-        usersyncInfo = createUsersyncInfo(Objects.requireNonNull(usersyncUrl), Objects.requireNonNull(platformId));
+    public BeachfrontUsersyncer(String usersyncUrl, String externalUrl, String platformId) {
+        usersyncInfo = createUsersyncInfo(Objects.requireNonNull(usersyncUrl), Objects.requireNonNull(externalUrl),
+                Objects.requireNonNull(platformId));
     }
 
     /**
      * Creates {@link UsersyncInfo} from usersyncUrl and platformId
      */
-    private static UsersyncInfo createUsersyncInfo(String usersyncUrl, String platformId) {
-        return UsersyncInfo.of(String.format("%s%s", usersyncUrl, platformId), "iframe", false);
+    private static UsersyncInfo createUsersyncInfo(String usersyncUrl, String externalUrl, String platformId) {
+        final String redirectUri = HttpUtil.encodeUrl(externalUrl)
+                + "%2Fsetuid%3Fbidder%3Dbeachfront%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}"
```

Comment:
The `{{.GDPR}}` and `{{.GDPRConsent}}` are Go-specific for templates.
We should use `{{gdpr}}` and `{{gdpr_consent}}` placeholders.